src/select/index.tsx

```diff
@@ -157,6 +158,8 @@ export const Select = factory(function Select({
 							}
 						}
 
+						const valueOption = find(options, (option) => option.value === value);
```

Comment:
this is going to cause me a whole heap of fun when i add resources here.. thanks :(